Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/ConceptIDLookup.java

```diff
@@ -84,6 +88,14 @@
 
         if (entry != null) {
 
+            try {
+                addAlternativeIds(pathParts[lastIndex], entry);
+            } catch (IllegalAccessException | LuceneException e) {
+                return new ResponseEntity<String>(msg.getXML(xmlEntries), HttpStatus.BAD_REQUEST);
+            } catch (IndexerRunningException ire) {
+                return new ResponseEntity<String>(msg.getXML(xmlEntries), HttpStatus.CONFLICT);
```

Comment:
I don't think that's the appropriate error code here: 
"This code is used in situations where the user might be able to resolve the conflict and resubmit the request."

This is not true in this case. The user just has to wait until indexing is done. I'd send a ```503``` ("The 503 (Service Unavailable) status code indicates that the server is currently unable to handle the request due to a temporary overload or scheduled maintenance, which will likely be alleviated after some delay."). It's not perfect but works.